podpac/core/data/datasource.py

```diff
@@ -326,16 +370,19 @@ def _eval(self, coordinates, output=None):
                                                    coordinates)
 
         # get data from data source
-        self._requested_source_data = self._get_data(coordinates)
+        self._requested_source_data = self._get_data()
 
         # if not provided, create output using the evaluated coordinates, or
         # if provided, set the order of coordinates to match the output dims
+        # Note that at this point the coordinates are in the same CRS as the native_coordinates
         if output is None:
             requested_dims = None
             output = self.create_output_array(coordinates)
         else:
             requested_dims = coordinates.dims
             coordinates = coordinates.transpose(*output.dims)
+            # TODO: there is no real way to know the crs of an input output UnitsDataArray
```

Comment:
@mlshapiro So are there plans to add the crs to the UnitsDataArray metadata?

Reply:
yes, as of the meeting yesterday - I was planning to wait until @mpu-creare #228 is merged and then I will pursue this. I will make an issue for this. It may wait until after this release, but its a small change